### DISCUSSION TITLE
Changes & 🐛

### DISCUSSION THREAD

**Discussion by Posandu** [blueedgetechno/win11React#63] [Ideas]
first of all I Love this project !!! But, sadly there are few changes you might need to do
## With Changes
![image](https://user-images.githubusercontent.com/76736580/129746542-8c9e9277-37a1-45bb-9608-7dc9eaa758c4.png)
## Now
![image](https://user-images.githubusercontent.com/76736580/129746954-c2497151-33e9-4c77-b4a7-a95ca4c428d1.png)

> Fonts are not clear
> Desktop icons can be optimized 
> Add dark mode
> Taskbar icons are not the same

That's all : ) 😀

**Comment by blueedgetechno** [comment]
1. what exactly do you mean by "Fonts are not clear"?
2. I would love to hear how!
3. Oh boy, it is challenging (but not for me, or ... maybe .. it is ... hmm)
3. I tried my best, if you have some please care to share.

**Reply by Posandu** [reply]
I mean the fonts are too small. Because of that they look blurlly.

**Reply by Posandu** [reply]
And it's hard to edit the css because there are so many of them ```inlined``` :(

**Reply by blueedgetechno** [reply]
I know pal, The fonts are small but they are meant to be small and browsers has its limitation. You have to compromise between performance or quality.
